### PR TITLE
Fill info on Python programming language

### PR DESCRIPTION
**Pull request by anxolerd**
- Add linters
- Add PyCharm IDE

### CODE REVIEW

**Comment by anxolerd** [comment]
P.S. I'm not sure where to state CPython, as it is not a compiler, but an interpreter. Or should I replace `Compilers` section with the `Interpreters` one?

**Comment by tshemsedinov** [comment]
@anxolerd tihis table of contents is copypasted for all languages, feel free to change doc structure

**Comment by aqrln** [comment]
@tshemsedinov well, there are no strong reasons to revert the merge, so let it be, but what's point in merging an incomplete work-in-progress PR right after answering a question about how to complete it the right way? 😄

File: Python/README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 Required software:
   1. Compilers:
-  2. Linters:
+  2. Linters: [flake8](http://flake8.pycqa.org/), [pyflakes](https://github.com/PyCQA/pyflakes), [mypy](http://mypy-lang.org/) *(experimental)*
```
Comment (aqrln) [inline Python/README.md:8]: @anxolerd I wouldn't say that mypy is a linter. Linters only check the code style while **static type checkers** for dynamically typed languages present a completely different sort of development tools that is certainly missing in the list. I'd prefer to add such entry for Python and JavaScript, with the examples of mypy and Flow respectively.
@tshemsedinov, what do you think about it?

Comment (tshemsedinov) [reply Python/README.md:8]: That just to make it visible to others.
What about Flow, I think we may add section: "Type checkers" or place it in "Other tools" section.

**Pull request merged**